optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -72,12 +77,12 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
             } else {
                 standardDeviationString.append("/");
             }
-            String abbreviated = Double.toString(standardDeviationDouble);
-            // Abbreviate to 2 decimals
-            // We don't use DecimalFormat to abbreviate because it's written locale insensitive (like java literals)
-            int dotIndex = abbreviated.lastIndexOf('.');
-            if (dotIndex >= 0 && dotIndex + 3 < abbreviated.length()) {
-                abbreviated = abbreviated.substring(0, dotIndex + 3);
+            // See http://docs.oracle.com/javase/7/docs/api/java/lang/Double.html#toString%28double%29
+            String abbreviated;
+            if (0.001d <= standardDeviationDouble && standardDeviationDouble <= 10000000d) {
```

Comment:
PS: That lowercase L problem is a famous java puzzler (I forgot which year):
  https://www.parleys.com/tutorial/the-java-puzzlers
  https://www.parleys.com/tutorial/java-puzzlers-scraping-bottom-barrel